src/packages/core/workspace/contexts/submittable-workspace-context-base.ts

```diff
@@ -46,6 +50,16 @@ export abstract class UmbSubmittableWorkspaceContextBase<WorkspaceDataModelType>
 		this.consumeContext(UMB_MODAL_CONTEXT, (context) => {
 			(this.modalContext as UmbModalContext) = context;
 		});
+
+		// TODO: Temporary keyboard shortcut solution, should be changed to a better generic shortcut solution. [NL]
+		(this.getHostElement() as HTMLElement).addEventListener('keyup', (e: KeyboardEvent) => {
```

Comment:
Experimenting, I had more success with using `'keydown'` to prevent the browser's "Save as..." dialog.

Reply:
I tried binding the event listener to the `document` and had greater success, but that led to a few other issues, so I'm not suggesting that we do that, just noting that the browser's "Save as..." dialog is preventable at the top of the DOM.